source/logs.txt

```diff
@@ -12,8 +12,11 @@ Retrieve Shell Logs
    :depth: 1
    :class: singlecol
 
-|mdb-shell| stores logs for each session in `ndjson
-<http://ndjson.org/>`__ format. 
+Starting in :binary:`mongosh` version 1.0.5, |mdb-shell| stores logs
+for each session in the same format as the MongoDB server logs. Earlier
+versions of :binary:`mongosh` used the `ndjson <http://ndjson.org/>`__
+format. For details on the current log formats, see
```

Comment:
But the server logs also use ndjson, i.e. the existing text was valid here

It might be better to only say that, before 1.0.5, mongosh uses a nonstandard log format

Reply:
Done.